### DISCUSSION TITLE
Create an area to view and manage all your shared notes in one place

### DISCUSSION THREAD

**Discussion by alangrainger** [alangrainger/share-note#27] [Vote for upcoming features]
This will probably be part of the plugin in the future, but for now you can use Dataview to achieve this quite simply and effectively:

````
```dataview
TABLE WITHOUT ID
  link(file.path, truncate(file.name, 28)) as Note,
  dateformat(share_updated, "yyyy-MM-dd") as "Shared on", 
  elink(share_link, regexreplace(share_link, "^.*?(\w+)(#.+?|)$", "$1")) as Link,
  choice(regextest("#", share_link), "🔒", "") as ""
WHERE share_link
```
````

To sort with your most recent shares at the top, add `SORT share_updated DESC` under the WHERE line.

![](https://docs.note.sx/notes/note-management.png)

I have also added this to the docs, and will keep it updated here:

https://docs.note.sx/Notes/Managing-your-notes

**Comment by ksdavidc** [comment]
I vote for this. :-)

**Comment by ksdavidc** [comment]
I renamed a note and it ended up with a new share url, which now leaves me with 2 shared notes. I was hoping it would just rename the note. Did I do something wrong, or is this how it is supposed to work?

**Comment by alangrainger** [comment]
@ksdavidc you can rename notes as much as you want and it won't affect the share URL.

As long as you don't delete the share URL it will keep re-using the same one. Please create a test note, share it, rename it then share it again, and you should see that the URL remains the same.

**Comment by alangrainger** [comment]
Deleting notes is now added in v0.6.0:

Click the 🗑️ icon next to the shared link to delete, or use the `Delete note` command from the command palette:

<img src="https://github.com/alangrainger/obsidian-share/assets/16197738/ec22623f-3d75-4f9d-9eca-51f73d3eb061" width="320">

**Reply by ksdavidc** [reply]
YAY!! This is very handy now for quick temprorary sharing of notes. It is doing a good job of copying my style sheet, which I like. Thanks.

**Comment by stojan97** [comment]
I would propose simple exporter of some HTML file or multiple files with a simple GUI (nagivation menu, title + logo, table of contents) would be great.

It would be great like this, and top of that even better would be to automate the deployment of those statistically generated files. 

Allow users to publish them on any platform of their choice (Netlify, Vercel)

There are some options that have tried this ([Obsidian Digital Garden](https://github.com/oleeskild/obsidian-digital-garden), [Obsidian Webpage export](https://github.com/KosmosisDire/obsidian-webpage-export)) but both of them are not as good as this one in **converting** the Obsidian documents to HTML page.

**Comment by ksdavidc** [comment]
While waiting for this to be developed, I created a "[data-loom](https://github.com/trey-wallis/obsidian-dataloom)" filtered on the YAML fields to keep track of all the shares in my vault. This can give the file and links, but of course it would be nice to be able to delete files as well from the loom.
<img width="955" alt="Screenshot 2024-04-07 at 11 58 23 AM" src="https://github.com/alangrainger/share-note/assets/62230004/6f7aec48-7990-43e6-88db-cacdb341176a">

Dataloom is not neccessary, but was easy (and I found a bug in the plugin along the way), and one could use dataview or other way to generate the table.

**Comment by ferileho** [comment]
You can simple use Saves Search... to Bookmarks

![image](https://github.com/alangrainger/share-note/assets/46302883/6a5e495a-9fc3-419b-9676-9669444adb36)

**Comment by alangrainger** [comment]
@ksdavidc @ferileho those are both such excellent and simple suggestions! I have [updated the docs](https://docs.note.sx/notes/management) with a nice Dataview query which will give you a view of all your shared notes, along with the date, the public link, and whether it's encrypted or not.

![](https://docs.note.sx/notes/note-management.png)

**Comment by poetex** [comment]
Is this issue still in work? Per the documentation it appears that this has been implemented, at least at a logical level. Perhaps stylistic changes and further adaptations should be their own new feature requests.

https://docs.note.sx/notes/managing-your-notes

**Reply by ferileho** [reply]
best way now is to use plugin Dataview.

I personaly use this:

```dataview
table  file.tags, updated, share_updated, share_link
WHERE share_link
SORT share_updated desc
```

**Reply by alangrainger** [reply]
Probably a good idea - I will close this for now.
